src/VisualStudio/Core/Test.Next/Services/AssetStorageTests.cs

```diff
@@ -34,7 +36,7 @@ public void TestGetAssets()
         {
             var storage = new AssetStorage();
 
-            var checksum = new Checksum(Guid.NewGuid().ToByteArray());
+            var checksum = Checksum.Create(WellKnownSynchronizationKind.Null, ImmutableArray.CreateRange(Guid.NewGuid().ToByteArray()));
```

Comment:
Won't this double the garbage since we have to make the byte[] then copy it into a new byte[] inside ImmutableArray?  I think we used byte[] (even though it's mutable) to avoid that.